### PR TITLE
Fix Symbiotic's tool-info module: Deploy all components of the archive

### PR DESCRIPTION
**Pull request by dbeyer**
This PR fixes issue #698 for Symbiotic in order to successfully participate in SV-COMP 2021.

### CODE REVIEW

**Comment by dbeyer** [comment]
@PhilippWendler Unfortunately, I need a new tag for this version and re-run Symbiotic in SV-COMP 2021 with the new version. Perhaps 3.5.1 as a tag for the branch?

File: benchexec/tools/symbiotic4.py [hunk 1]
```diff
@@ -15,15 +15,7 @@ class Tool(benchexec.tools.template.BaseTool2):
...
-    REQUIRED_PATHS = [
-        "bin",
-        "include",
-        "share",
-        "instrumentations",
-        "lib",
-        "lib32",
-        "symbiotic",
-    ]
+    REQUIRED_PATHS = ["."]
```
Comment (mchalupa) [inline benchexec/tools/symbiotic4.py:18]: By the way, this could be a good default value of this attribute.

**Review by mchalupa — APPROVED** [review]
The change looks good, I tested with `benchexec.test_tool_info` and everything seems alright.

File: benchexec/tools/symbiotic4.py (hunk 1 above)
Comment (mchalupa) [reply benchexec/tools/symbiotic4.py:18]: Oh, that is exactly what is in the issue #698 :)

**Comment by PhilippWendler** [comment]
Ok, I will merge this and create release 3.6.

In the future, please do not start branches at exactly the tag revision, because then we have more than one commit in the repository where `benchexec --version` produces a number like `3.5`. I fixed this now by rebasing the branch on top of 473c0eaf904a4fecc16d36841dfd04daa2d2bb03.

**Comment by dbeyer** [comment]
> Ok, I will merge this and create release 3.6.

Thank you!

> rebasing the branch on top of [473c0ea](https://github.com/sosy-lab/benchexec/commit/473c0eaf904a4fecc16d36841dfd04daa2d2bb03).

But what if I do not want all those other commits? On the branch I was sure that I changed only the Symbiotic file and nothing else. Now I have all those other changes. I think they are not touching the code that I had executed for the competition, but what if there is a strange interference and then the results are not reproducible?  Shouldn't a bug-fix release be an end of a (not merged) branch, while the commits leading there are merged into the master later on (by cherry-picking?), and I can still take the branch with as few commits as possible?

**Comment by PhilippWendler** [comment]
Misunderstanding: I did not rebase the branch on top of yesterday's master, I rebased it on top of the linked commit, which is exactly 1 commit after the tag 3.5 and which contains only a changed version number. So the bug fix release contains exactly what it should contain.

**Comment by dbeyer** [comment]
Indeed!  Very nice.  Thanks a lot!

**Pull request merged**